question.py

```diff
@@ -72,19 +69,43 @@ def create_tags(app_uuid, user_id, tags):
                    text_uuid=tag["text_uuid"],
                    score=tag["tag"])
         all_tags = list(Tag.select().where(Tag.text_uuid == tag["text_uuid"]))
-        if len(all_tags) >= MIN_QUESTIONS:
-            all_scores = [t.score for t in all_tags]
-            # It is tempting to use standard deviation, but it is not sensitive
-            # to outliers. e.g. stdev([1, -0.5, -0.6]) ~= 0.9
-            if min_diff(all_scores) <= MAX_DIFF:
+
+        # I shall only consider texts tagged more than MIN_TAGS times
+        # as candidadtes for complete texts
+        # Whether they are completed depends on the error margin given by those 
+        # tags
+        if len(all_tags) >= MIN_TAGS:
+            mean = statistics.mean([tag.score for tag in all_tags])
+            sigma = statistics.stdev([tag.score for tag in all_tags])
+            error = 1.96 * sigma / sqrt(len(all_tags))
```

Comment:
Why 1.96?

Reply:
https://en.wikipedia.org/wiki/1.96
I'm still changing this to use a t-student distribution (It's supposed to be better for small samples)